### PR TITLE
Adding fromBase16 to the frontend

### PR DESCRIPTION
**Pull request by thub1271**
Adding fromBase16 to the frontend

### CODE REVIEW

File: sigmastate/src/test/scala/sigmastate/lang/SigmaParserTest.scala [hunk 1]
```diff
@@ -603,6 +603,7 @@ class SigmaParserTest extends PropSpec with PropertyChecks with Matchers with La
   }
 
   property("fromBaseX string decoding") {
+    parse("""fromBase58("111")""") shouldBe Apply(FromBase16Func.symNoType, IndexedSeq(StringConstant("111")))
```
Comment (aslesarenko) [inline sigmastate/src/test/scala/sigmastate/lang/SigmaParserTest.scala:606]:
```suggestion
    parse("""fromBase16("111")""") shouldBe Apply(FromBase16Func.symNoType, IndexedSeq(StringConstant("111")))
```

**Comment by aslesarenko** [comment]
@thub1271 looks good, please fix the test

**Comment by thub1271** [comment]
@aslesarenko I have fixed the test. Please review when you get a chance.

**Comment by kushti** [comment]
@thub1271 are you on Discord ? https://discord.gg/kj7s7nb dev chats are there usually, over #development channel

**Review by aslesarenko — APPROVED** [review]

**Review by aslesarenko — CHANGES_REQUESTED** [review]
The tests need to be fixed

**Comment by aslesarenko** [comment]
@jozanek please also add this function to LangSpec.md

**Review by aslesarenko — APPROVED** [review]

**Pull request merged**